flocker/provision/_install.py

```diff
@@ -156,6 +156,21 @@ def task_install_kernel_devel():
 """)])
 
 
+def task_disable_selinux():
+    """
+    Disable SELinux for this session and permanently.
+    XXX: Remove this when we work out suitable SELinux settings.
+    See https://clusterhq.atlassian.net/browse/FLOC-619.
+    """
+    return sequence([
+        run("if selinuxenabled; then setenforce 0; fi"),
+        run("test -e /etc/selinux/config && "
+            "sed --in-place='.preflocker' "
+            "'s/^SELINUX=.*$/SELINUX=disabled/g' "
```

Comment:
We should probably set this to permissive, rather than disabled. This will cause errors to be logged (but not prevented). This will be useful when we decide to properly address selinux support.